carbonmark-data/app/[locale]/retirement-trends/styles.module.scss

```diff
@@ -0,0 +1,26 @@
+@import "~@klimadao/lib/theme/breakpoints.scss";
+
+.tabButton {
+  font-family: Poppins;
+  font-weight: 700;
+  font-size: 1.6rem;
+}
+
+.mobileOnly {
+  @include large {
+    display: none;
+  }
+}
+
+.desktopOnly {
+  display: none;
+
+  @include large {
+    display: unset;
+  }
+}
```

Comment:
Just a note: Maybe we can put `.mobileOnly` and `.desktopOnly` in `theme/layout.module.scss` later. I feel like we will reuse that.

Reply:
Done in [c4cd34a](https://github.com/KlimaDAO/klimadao/pull/1518/commits/c4cd34a7d2959a69ce27d2632d04ca21de2a4faa)